### PR TITLE
Pensions e2e additional ceh

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3741

### CODE REVIEW

File: test/pensions/00-all-fields.e2e.spec.js [hunk 1]
```diff
@@ -152,8 +152,150 @@ const runTest = E2eHelpers.createE2eTest(
...
+    E2eHelpers.expectNavigateAwayFrom(client, '/financial-disclosure/otherExpenses/dependents/1');
 
     // Additional Information page
+    // direct deposit
+    client.waitForElementVisible('label[for="root_view:noDirectDeposit"]', Timeouts.normal);
+    client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(1)', 'progress-segment-complete');
+    PageHelpers.completeDirectDepositInfo(client, testData.data);
+    client.axeCheck('.main')
+      .click('.form-panel .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/direct-deposit');
```
Comment (cehsu) [inline test/pensions/00-all-fields.e2e.spec.js]: Should we replace `personal` with `additional` here?

Comment (jbalboni) [inline test/pensions/00-all-fields.e2e.spec.js]: Yeah, I think that makes sense.

**Review by jbalboni — APPROVED** [review]
Looks fine, just make sure to look at my comments on the other PR before merging this, since it includes those changes, too.

File: test/e2e/pensions-helpers.js [hunk 2]
```diff
@@ -170,6 +170,116 @@ function completeDependentAddressInfo(client, data, index) {
   }
 }
 
+function completeNetWorthInfo(client, data) {
```
Comment (cvalarida) [inline test/e2e/pensions-helpers.js]: Some of these functions are also in [your previous PR](https://github.com/department-of-veterans-affairs/vets-website/pull/6079/files), by the by. I'm assuming you branched off that other branch, so you can either do some git magic to remove first two commits (but don't ask me how!), or deal with the merge conflict later.

Don't make no nevermind to me either way! Just thought I'd do my civic duty and point it out.

**Review by cvalarida — APPROVED** [review]

**Pull request merged**